libs/community/langchain_community/document_loaders/notiondb.py

```diff
@@ -20,13 +20,17 @@ class NotionDBLoader(BaseLoader):
         database_id (str): Notion database id.
         request_timeout_sec (int): Timeout for Notion requests in seconds.
             Defaults to 10.
+        filter_object (Dict[str, Any]): Filter object used to limit returned
+            entries based on specified criteria.
+            Defaults to empty dict, which will return ALL entries.
     """
 
     def __init__(
         self,
         integration_token: str,
         database_id: str,
         request_timeout_sec: Optional[int] = 10,
+        filter_object: Optional[Dict[str, Any]] = {}
```

Comment:
* New named arguments should be introduced after a `*` operator.
* `{}` is mutable

```suggestion
        *,
        filter_object: Optional[Dict[str, Any]] = None
```